module.json

```diff
@@ -11,6 +11,10 @@
       {
         "name": "Kandashi",
         "url": "https://github.com/kandashi"
+      },
+      {
```

Comment:
For now, remove this author line. 1.8.5 will be credited to you.

We'll find a time to sit down with you and talk about expanding the dev team 😃